PRIVACY.md

```diff
@@ -35,23 +36,40 @@ To configure metrics, pass the `--metrics` option to Semgrep:
 - `--metrics on`: metrics are sent on every Semgrep run
 - `--metrics off`: metrics are never sent
 
-Instead of the `--metrics` option, collection can also be controlled by setting the `SEMGREP_SEND_METRICS`
-environment variable to any of `auto`, `on`, or `off`.
+Alternatively, set the `SEMGREP_SEND_METRICS` environment variable to `auto`, `on`, or `off`.
 
-Note that certain Semgrep integrators turn on metrics for every run. For example, both the [Semgrep CI agent](https://github.com/returntocorp/semgrep-action) and [GitLab's Semgrep SAST analyzer](https://gitlab.com/gitlab-org/security-products/analyzers/semgrep) use `--metrics on` by default.
+Note that certain Semgrep integrators turn on metrics for every run.
+For example, [GitLab's Semgrep SAST analyzer](https://gitlab.com/gitlab-org/security-products/analyzers/semgrep) uses `--metrics on` by default.
 
 ## Data NOT collected
 
-We strive to balance our desire to collect data for improving Semgrep with our users' need for privacy and security. After all, we are a security tool! The following never leave your environment and are not sent or shared with anyone.
+### Data NOT collected ever
+
+We strive to balance our desire to collect data for improving Semgrep
+with our users' need for privacy and security.
+After all, we are a security tool!
+The following never leave your environment and are not sent or shared with anyone.
 
 - Source code
-- Filenames, file contents, or commit hashes
-- User-identifiable data about Semgrep’s findings in your code, including finding messages
 - Private rules
 
-## Data collected
+### Data NOT collected unless explicitly requested
+
+The following data will never leave your environment as part of metrics.
+
+- Filenames, file contents
+- Git commit hashes, messages, authors
+- User-identifiable data about Semgrep’s findings in your code, including finding messages
+
+This data will be sent to Semgrep App only if you explicitly request it
+(such as with `semgrep login && semgrep ci` to connect with Semgrep App,
+or with the upcoming `semgrep flare` to send a report to support)
+Even in that case, your source code and private rules will never be sent.
```

Comment:
```suggestion
This data will be sent to Semgrep App only if you explicitly request it (such as with `semgrep login && semgrep ci` to connect with Semgrep App, or with the upcoming `semgrep flare` to send a report to support). Even in that case, your source code and private rules will never be sent.
```

Also you could exclude the mention of `semgrep flare` since it’s not available yet. If you leave it in, you’ll still need to come back here and update this text because it will no longer be “upcoming.”

Reply:
@pabloest yeah, don't you think it clarifies what we mean by "explicitly request" though? If it improves understanding, I'd think it's worth it to leave the "upcoming" here and have to come back later.

Reply:
Sort of. It does help clarify but it also introduces new questions about what semgrep flare is. How about something like this? 

> This data will be sent to Semgrep App only if you explicitly request it, such as with `semgrep login && semgrep ci` to connect with Semgrep App or by sending a report to support.

Reply:
I ended up removing the whole debug report section, thank you!